### ISSUE TITLE
Problem when referenced issue is not found

### ISSUE THREAD

**Issue by vhkrausser** [lerna/lerna-changelog#77]
I'm facing some issues while testing your lib. In the line referenced, you try to map the issues labels.
For some reasons I get in the github response 

```JSON
{ 
    "commitSHA":"xxxxxx",
    "message":"Merge pull request #838 from xxxx/dev",
    "tags":["vxxxx"],
    "issueNumber":"838",
    "date":"2017-09-14",
    "githubIssue":{
        "message":"Not Found",
        "documentation_url":"https://developer.github.com/v3"
    }
}
```

https://github.com/lerna/lerna-changelog/blob/a41328748b472b59651e235607f3274fa319c1ae/src/changelog.ts#L195

**Comment by vhkrausser** [comment]
Found my problem.

This occurs when you do not inform a repo owner/name in the configuration.

Searching I didn't found any place that inferred this if not informed.

**Comment by Turbo87** [comment]
This should also be fixed now in #82
